### PR TITLE
IACO-679: Follow on feedback - Android

### PR DESCRIPTION
**Pull request by grozen**
This pull requests contains changes in light of the feedback gathered during the iOS party.

### CODE REVIEW

File: doc/integration.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ dependencies {
 ##Supplying your API key
 In order to use the SDK, you will need an API key to identify yourself. You can get one from our [developer site (coming soon)](http://developers.klarna.com/).
 
-We recommend setting your API key in your main activity in the manner shown below:
+We recommend setting your API key in your main activity in the manner shown below. If you don't have a key of your own yet the one listed below will work as well but will not properly represent your application:
```
Comment (tomerlevi444) [inline doc/integration.md]: `work as well but` -> `work as well, but`

**Comment by tomerlevi444** [comment]
Approved!

File: doc/integration.md [hunk 2]
```diff
@@ -127,38 +129,26 @@ Note the following:
...
+While we've seen how to utilize the registration view, we never talked about **when** you should display it. Our recommendation is to display the registration view when you do not have a user token stored. Assuming your user has gone through the registration process successfully and received a token there is no need to have the user register again, as tokens do not expire (though they can be revoked).
 
 ##Performing purchases
 The aim of this SDK is to allow users to make purchases using your application, backed by Klarna as a payment method. However, the SDK does not offer any direct methods for performing purchases as this will expose your application's private Klarna credentials. Instead, applications using the SDK are expected to work in concert with an application backend, which will perform the actual purchase requests.
```
Comment (yuval-netanel) [inline doc/integration.md]: I think it's better to drop "in concert", just:
Instead, applications using the SDK are expected to work with an application backend, which will perform the actual purchase requests.

**Comment by yuval-netanel** [comment]
Approved!

File: doc/integration.md [hunk 3]
```diff
@@ -33,10 +33,21 @@ dependencies {
 }
 ```
 
+###Granting internet access
+To function properly, the SDK requires access to the internet. You will need to make sure that the following user permission appears in your application's manifest:
+
+```xml
+<manifest xlmns:android="http://schemas.android.com/apk/res/android">
+  ...
+  <uses-permission android:name="android.permission.INTERNET" />
```
Comment (yuval-netanel) [inline doc/integration.md]: I think it's better to add this to our AndroidManifest.xml

Comment (MaTriXy) [inline doc/integration.md]: I think  Best usage is add all permissions needed to Sample app and show entire project with relevant permissions.

**Pull request merged**